boa/src/builtins/console.rs

```diff
@@ -32,11 +32,36 @@ pub fn error(_: &Value, args: &[Value], _: &mut Interpreter) -> ResultValue {
     Ok(Gc::new(ValueData::Undefined))
 }
 
+/// Print a javascript value to the standard error if first argument evaluates to false or there were no arguments
+/// 
+/// https://console.spec.whatwg.org/#assert
+pub fn assert(_: &Value, args: &[Value], _: &mut Interpreter) -> ResultValue {
+    let assertion = if !args.is_empty() {
+        from_value::<bool>(args[0].clone()).expect("Could not convert to bool.")
+    } else {
+        false
+    };
+
+    if !assertion {
+        eprint!("Assertion failed: ");
+        for (i, item) in args.iter().skip(1).enumerate() {
+            eprint!("{}", item);
+            if i + 1 != args.len() {
+                eprint!(" ");
+            }
+        }
```

Comment:
I think we can do this even easier to understand :)
```suggestion
        eprint!("Assertion failed:");
        for message in args.iter().skip(1) {
            eprint!(" {}", message);
        }
```

Reply:
:)